client/src/i18n/fr/report.json

```diff
@@ -190,7 +190,10 @@
       "TOTAL_ENTRIES": "Total des entrées",
       "TOTAL_EXITS": "Total des sorties",
       "TOTAL_QUANTITIES_CONSUMED": "Total consommées",
-      "TOTAL_QUANTITIES_LOST": "Total perdues"
+      "TOTAL_QUANTITIES_LOST": "Total perdues",
+      "TOTAL_DAYS_OF_STOCK_OUT" : "Nombre total de jours de rupture de stock",
+      "TOTAL_DAYS" : "Nombre total de jours",
```

Comment:
Since it is related to stock and not number of days in month, you could say: "Nombre total de jours de stock".